server/api/initial_data.py

```diff
@@ -139,7 +139,7 @@ def init_data(
 data_version_prior_to_table_addition = 1
 
 # NOTE: Bump this number when adding a new data migration
-latest_data_version = 7
+latest_data_version = 8
```

Comment:
Why 8?
We have only 7:
https://github.com/mlrun/mlrun/blob/1d7113436fe7c83b5eb619b2cc12a304acac352a/server/api/initial_data.py#L231-L247

and it is already bumped on this pr:
https://github.com/mlrun/mlrun/pull/6129

Reply:
@roei3000b https://github.com/mlrun/mlrun/pull/6389